scripts/_helpers.py

```diff
@@ -821,3 +818,68 @@ def get_last_commit_message(path):
 
     os.chdir(backup_cwd)
     return last_commit_message
+
+
+def get_dirname_abs_path(path):
+    """
+    It returns the directory name of a normalized and absolutized version of
+    the path.
+    """
+    return str(pathlib.Path(path).absolute().parent)
+
+
+def get_dirname_path(path):
+    """
+    It returns the directory name of the path.
+    """
+    return str(pathlib.Path(path).parent)
+
+
+def get_abs_path(path):
+    """
+    It returns the absolutized version of the path.
+    """
+    return str(pathlib.Path(path).absolute())
+
+
+def get_basename_abs_path(path):
+    """
+    It returns the base name of a normalized and absolutized version of the
+    path.
+    """
+    return pathlib.Path(path).absolute().name
+
+
+def get_basename_path(path):
+    """
+    It returns the base name of the path.
+    """
+    return pathlib.Path(path).name
+
+
+def get_path(*args):
+    """
+    It returns a new path string.
+    """
+    return str(pathlib.Path(*args))
+
+
+def get_posixpath(*args):
+    """
+    It returns a new PosixPath object.
+    """
+    return pathlib.Path(*args)
+
+
+def get_path_size(path):
+    """
+    It returns the size of a path (in bytes)
+    """
+    return pathlib.Path(path).stat().st_size
+
+
+def build_directory(path):
+    """
+    It creates recursively the directory and its leaf directories.
+    """
+    pathlib.Path(path).mkdir(parents=True, exist_ok=True)
```

Comment:
For consistency, just tagging that abstractions are missing for: 

- `str(pathlib.Path.cwd())`

- `str(pathlib.Path(str(config["destination"])).relative_to("."))` 

- `pathlib.Path(base_folder, "configs").exists()`

Maybe also a few more.